projects/elife.yaml

```diff
@@ -551,6 +551,8 @@ journal:
                     - "original-host"
                     - "ping-status"
                     - "strip-non-journal-cookies"
+                    - "journal-google-scholar"
+                    - "journal-google-scholar-vary"
```

Comment:
the naming correctly reflects the specificity of the snipper for the `journal` project